mailpoet/lib/Migrations/Migration_20230503_210945.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Migrations;
+
+use MailPoet\Entities\SubscriberEntity;
+use MailPoet\Migrator\Migration;
+
+class Migration_20230503_210945 extends Migration {
+  public function run(): void {
+    $subscribersTable = $this->getTableName(SubscriberEntity::class);
+    $this->connection->executeQuery(
+      "ALTER TABLE `{$subscribersTable}`
+        ADD INDEX `first_name` (`first_name`),
+        ADD INDEX `last_name` (`last_name`)"
```

Comment:
These columns seem to be `varchar(255)`. We had some issues indexing those, right? I think with `utf8mb4` the maximum indexed column length is 191 (that's why we use it everywhere).

That said, we could just index a much smaller prefix anyway, what do you think? I mean, it could make the indexes huge, if we indexed those 191 maximum chars, and the benefit would probably be small. If most users will type a prefix of just a few chars, then indexing like first 10-30 chars could be enough.

Reply:
Thanks so much for catching this! I think an index length of 10 should be plenty for name columns so I'm going to go with that.